ui/push-health/PlatformConfig.jsx

```diff
@@ -56,11 +70,19 @@ class PlatformConfig extends React.PureComponent {
 
     return (
       <Row
-        className="ml-5 pt-2 mr-1"
+        className={`ml-5 pt-2 mr-1 ${isInvestigated && 'investigated'}`}
```

Comment:
I don't understand the need for:
&& 'investigated' ?  does isInvestigated have a string, not a boolean?

Reply:
This is a way only add that text (in this case the class of `investigated` if the `isInvestigated` value is set to true.  It's just a syntactic shortcut.